net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/forms/dialogEdit.xml

```diff
@@ -43,6 +43,31 @@
       <type>checkbox</type>
       <help><![CDATA[Use the ID to set the TTL Value of the packet (Original method) rather than DSCP, default is not ticked]]></help>
    </field>
+   <field>
+      <id>udpbroadcastrelay.allowcidr</id>
+      <label>Allowed CIDRs</label>
+      <style>tokenize</style>
+      <type>select_multiple</type>
+      <allownew>true</allownew>
+      <help><![CDATA[Only allow packets from a range of IP source addresses]]></help>
+      <advanced>true</advanced>
+   </field>
+   <field>
+      <id>udpbroadcastrelay.blockcidr</id>
+      <label>Blocked CIDRs</label>
+      <style>tokenize</style>
+      <type>select_multiple</type>
+      <allownew>true</allownew>
+      <help><![CDATA[Block packets from a range of IP source addresses<br/>Where overlapping CIDRs are specified with the Block/Allow CIDRs, the most specific match (longest prefix) will take effect]]></help>
+      <advanced>true</advanced>
+   </field>
+   <field>
+      <id>udpbroadcastrelay.msearch</id>
+      <label>M-SEARCH processing</label>
+      <type>text</type>
+      <help><![CDATA[A space-separated list of M-SEARCH processing actions, use a comma before search terms for example <code>block,search-term</code><br/>Please refer to the <a href="https://github.com/marjohn56/udpbroadcastrelay#usage">UDP Broadcast Relay documentation</a> for more information]]></help>
```

Comment:
HTML in these parts just ends up bouncing the help text from translations because the translation could break the HTML due to typos which are hard to spot and debug. We don't have a good solution for this.

Reply:
Did you want me to remove the link to the documentation? Otherwise not sure what I am meant to do for this one